src/app/loans/loans-view/working-capital/loan-breach-schedule-tab/loan-breach-schedule-tab.component.scss

```
@@ -0,0 +1,492 @@
+}
+
+.table-wrap {
+  overflow-x: auto;
+}
+
+.breach-table {
+  width: 100%;
+  background: var(--ch-surface);
+}
+
+:host ::ng-deep {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)